src/session/sidebar/row.rs

```diff
@@ -479,6 +487,16 @@ fn stringify_message(message: Message) -> String {
                 ChatType::Supergroup(supergroup) if supergroup.is_channel() => {
                     gettext("Channel photo removed")
                 }
+                ChatType::Supergroup(supergroup) if !(supergroup.is_channel()) => {
```

Comment:
Instead of creating another match arm you can add an else to the if of the previous match arm.